.travis.yml

```diff
@@ -7,6 +7,9 @@ node_js:
   - "lts/*"
   - "7"
   - "6"
-script:
-  - npm test
-  - yarn danger ci
+  - "5"
+  - "4"
+jobs:
```

Comment:
@jwu910 adding a test stage and the associated script should help you out with eventually choosing what you test on (4 vs 5 vs 6)